config/upgrade.go

```diff
@@ -26,9 +25,10 @@ func Upgrade(fs afero.Fs, configFile string) error {
 			return err
 		}
 
-		marshalled, err := json.MarshalIndent(c2, "", "  ")
+		//Should this marshal it to v2?
```

Comment:
what's this comment?

Reply:
That was a mental note that I forgot to remove